testutil/testaudit/docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: "3.5"
+services:
+  alpha1:
+    image: dgraph/dgraph:local
+    working_dir: /data/alpha1
+    ports:
+      - "8080"
+      - "9080"
+    volumes:
+      - type: bind
+        source: $GOPATH/bin
+        target: /gobin
+        read_only: true
+      - type: bind
```

Comment:
I still don't remember why can't we use volume instead of this

Reply:
I tried using a volume and ran into issues.  Then we tried debugging the problem together and couldn't figure it out.  The problem resolved when we switched back to using a bind mount.  We use bind mounts in the other audit log tests (that existed in the repo previously in `systest/audit`).  I don't want to introduce that kind of change here in this PR, because if we change it here we should also change it elsewhere too.